cli/lib/kontena/cli/nodes/ssh_command.rb

```diff
@@ -3,9 +3,9 @@ class SshCommand < Kontena::Command
     include Kontena::Cli::Common
     include Kontena::Cli::GridOptions
 
-    parameter "[NODE_ID]", "SSH to Grid node, or --any"
+    parameter "[NODE_ID]", "SSH to Grid node. Use --any to connect to the first available node"
```

Comment:
So commands can't be used with --any

A fix for this could be checking if `name == '--'`. Then `kontena node ssh --any -- echo foo` should work.